core/src/test/java/org/mobilitydata/gtfsvalidator/notice/OtherTestValidationNotice.java

```diff
@@ -0,0 +1,35 @@
+ /*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import java.util.Map;
+
+public class OtherTestValidationNotice extends ValidationNotice {
+
+  private final String code;
+
+  public OtherTestValidationNotice(
+      String code, Map<String, Object> context, SeverityLevel severityLevel) {
+    super(context, severityLevel);
+    this.code = code;
+  }
+
+  @Override
+  public String getCode() {
```

Comment:
I am unsure if you need that method. We plan to remove its implementations in child classes anyway.

Reply:
`getCode()` implementation is removed in https://github.com/MobilityData/gtfs-validator/pull/790 - but this PR has not been merged yet. I agree that in the future it should be removed.